### DISCUSSION TITLE
Keyboard Profiles for individual keyboard statistics

### DISCUSSION THREAD

**Discussion by DwieDave** [monkeytypegame/monkeytype#1370] [Ideas] [answered]
Hello!

I think named keyboard profiles to get differentiated 
statistics for individual keyboards would be a really nice feature.

Different Profiles could be edited in settings 
and quickly selected via the esc-menu, 

Many enthusiast keyboard users have multiple keyboards 
and maybe they would like to track and compare their 
individual typing speeds.

Maybe there are more people out here 
who would like to see this feature;
 
Please feel free to comment if you 
would like this feataure as well ! :)

Kind regards
DwieDave

**Comment by SeerLite** [comment, marked as answer]
Hi! Monkeytype already has this feature, it's called "tags". Look for them in the settings and account stats screens.

**Reply by DwieDave** [reply]
Thank you and sorry I didn't see the feature before!
